packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -362,7 +362,7 @@ export class WorkflowsController {
 
 		const workflow = await this.workflowService.delete(req.user, workflowId);
 		if (!workflow) {
-			this.logger.verbose('User attempted to delete a workflow without permissions', {
+			this.logger.warn('User attempted to delete a workflow without permissions', {
 				workflowId,
 				userId: req.user.id,
 			});
```

Comment:
Side note: The error should be `ForbiddenError` and not `BadRequestError`. No need to fix now, just pointing out